spec/system/partner_system_spec.rb

```diff
@@ -147,27 +152,28 @@
         let!(:partner_to_request_recertification) { create(:partner, status: 'approved') }
 
         before do
-          sign_in(@user)
-          visit partners_path(@organization)
+          sign_in(user)
+          visit partners_path(organization)
         end
 
         it 'should notify the user that its been successful and change the partner status' do
           accept_confirm do
             find_button('Request Recertification').click
           end
 
-          assert page.has_content? "#{partner_to_request_recertification.name} recertification successfully requested!"
+          expect(page).to have_content "#{partner_to_request_recertification.name} recertification successfully requested!"
           expect(partner_to_request_recertification.reload.recertification_required?).to eq(true)
         end
       end
     end
 
     describe "#index" do
       before(:each) do
-        @uninvited = create(:partner, name: "Bcd", status: :uninvited)
-        @invited = create(:partner, name: "Abc", status: :invited)
-        @approved = create(:partner, :approved, name: "Cde", status: :approved)
-        @deactivated = create(:partner, name: "Def", status: :deactivated)
+        partner
```

Comment:
Should we just replace the let(:partner) with `let!(:partner)` ?

Reply:
Honestly. If means of the test is unchanged I would rather remove partner and change the test to expect 4 partners instead of 5. It expects 5 only because in the past `@partner` was per seeded.

But either works.

Reply:
Up to you!